torch/_inductor/codecache.py

```diff
@@ -1513,24 +1526,38 @@ def get_include_and_linking_paths(
 
     # third party libs
     if config.is_fbcode():
-        ipaths.append(build_paths.sleef())
-        ipaths.append(build_paths.openmp())
-        ipaths.append(build_paths.python())
-        ipaths.append(build_paths.cc_include())
-        ipaths.append(build_paths.libgcc())
-        ipaths.append(build_paths.libgcc_arch())
-        ipaths.append(build_paths.libgcc_backward())
-        ipaths.append(build_paths.glibc())
-        ipaths.append(build_paths.linux_kernel())
-        ipaths.append(build_paths.cuda())
+        # Note that the order of include paths do matter, as a result
+        # we need to put each entire -I list despite the redundancies
+        if torch.version.hip is not None:
+            ipaths.append(build_paths.openmp())
+            ipaths.append(build_paths.python())
```

Comment:
nit - can we put the common ones like ```openmp```, ```python``` to the outside of ```if-else``` branches? Thanks.

Reply:
@chenyang78 Updated, please take another look. Since the order of include paths do matter, so a result, we need to have several branches interleaved here.